src/ui/public/agg_response/point_series/_ordered_date_axis.js

```diff
@@ -20,24 +20,19 @@
 import moment from 'moment';
 
 export function orderedDateAxis(chart) {
-  const xAgg = chart.aspects.x.aggConfig;
-  const buckets = xAgg.buckets;
-  const format = buckets.getScaledDateFormat();
+  const x = chart.aspects.x[0];
+  const format = x.params.format;
+  const bounds = x.params.bounds;
 
   chart.xAxisFormatter = function (val) {
     return moment(val).format(format);
   };
 
-  chart.ordered = {
-    date: true,
-    interval: buckets.getInterval(),
-  };
+  chart.ordered.date = true;
 
-  const axisOnTimeField = xAgg.fieldIsTimeField();
-  const bounds = buckets.getBounds();
-  if (bounds && axisOnTimeField) {
-    chart.ordered.min = bounds.min;
-    chart.ordered.max = bounds.max;
+  if (bounds) {
+    chart.ordered.min = isNaN(bounds.min) ? Date.parse(bounds.min) : bounds.min;
+    chart.ordered.max = isNaN(bounds.max) ? Date.parse(bounds.max) : bounds.max;
```

Comment:
yeah, i think its right .... boubds.min could either be a number (isNaN will return false), in which case we just take this number .... it can also be a date string, in which case we need to Date.parse it.

Reply:
Ah, of course. `Date.parse()` is taking a string. Makes sense.